CMSgov/easi-app
# [EASI-4125] System search bookmark side effects

# EASI-4125, EASI-4126


## Changes and Description

- EASI-4125 - Fixed systems table search results resetting when bookmarking a system
- EASI-4126 - Fixed systems table search results resetting when changing table view
- Fixed console errors from invalid nesting in `Alert` component



## How to test this change

To see the systems table search box with seeded data, comment out the conditional logic around `GlobalClientFilter` [here](https://github.com/CMSgov/easi-app/blob/de394c0696c51a98aa5e188025cade0863a984f7/src/views/SystemList/Table.tsx#L317).

1. Go to /systems?table-type=all-systems and scroll down to the table
2. If the search box is hidden, comment out the [conditional logic](https://github.com/CMSgov/easi-app/blob/de394c0696c51a98aa5e188025cade0863a984f7/src/views/SystemList/Table.tsx#L317) around `GlobalClientFilter`
3. Type "funny"
4. Click the bookmark icon next to "Office of Funny Walks"
5. Search term and results should persist
6. Click "My systems" button to change table view
7. Search term and results should persist



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
